tck-dist/src/main/asciidoc/concurrency-tck-reference-guide.adoc

```diff
@@ -0,0 +1,625 @@
+= Jakarta Concurrency TCK Reference Guide
+:author: Kyle Aure
+:email: kyle.aure@ibm.com
+:revnumber: v1.0
+:toc:
+:sectnums:
+
+:APIShortName: Concurrency
+:APIShortNameLC: concurrency
+:APILongName: Jakarta Concurrency
+:APIVersion: 3.0.0
+:APISpecSite: https://jakarta.ee/specifications/concurrency/3.0/
+:APIEclipseSite: https://projects.eclipse.org/projects/ee4j.cu
+:APIGitSite: https://github.com/jakartaee/concurrency
+:TCKTestPlatform: TestNG
+:RuntimeTestCount: 164
+:SigPluginGAV: org.netbeans.tools:sigtest-maven-plugin:1.6
+
+== Preface
+
+This guide describes how to download, install, configure, and run the Technology Compatibility Kit (TCK) used to verify the compatibility of an implementation of the {APILongName} specification.  
+
+The specification describes the job specification language, Java programming model, and runtime environment for {APILongName} applications.
+
+=== Licensing
+
+The {APILongName} TCK is provided under the *Eclipse Foundation Technology Compatibility Kit License - v 1.0* [https://www.eclipse.org/legal/tck.php].
+
+=== Who Should Use This Guide
+
+This guide will assist in running the test suite verifies implementation compatibility for:
+
+* implementers of the entire Jakarta EE Platform (of which {APILongName} is one component). 
+
+=== Terminology - "SE mode" vs. "EE mode"
+
+Building on the previous point, it is convenient to use, as shorthand, the term "EE mode" when talking about the TCK constructs and requirements specifically for users running the TCK to certify against the entire EE platform. It is a convenient shorthand term too, then, to use the term "SE mode" for users that are only trying to certify against the {APILongName} specification, though this term in some ways might be misleading.  
+
+Some specifications have a subset of tests that can run in "SE mode" without the requirement of running against an entire EE Platform.  The {APIShortName} TCK, however, runs all tests in "EE mode" and will require a full EE platform to test against.
+
+=== Terminology - "Standalone TCK"
```

Comment:
I don't know how pervasive this terminology will be throughout the document, but it would be more clear if we call it a "Specification TCK" or "Specification Project TCK" or something like that.  Intuitively, the term Standalone implies that the TCK runs on its own against Java SE without requiring an application server.